plugins/auth-backend/src/providers/config.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const providers = [
+  {
+    provider: 'google',
+    options: {
+      clientID: process.env.GOOGLE_CLIENT_ID!,
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET!,
```

Comment:
we probably want to namespace these env vars [like lighthouse does](https://github.com/spotify/lighthouse-audit-service/blob/master/src/run.ts#L21), so they don't collide between plugins

Reply:
Agree. Unable to decide between `AB_GOOGLE_CLIENT_ID` or `ABS_GOOGLE_CLIENT_ID` or `BACKSTAGE_GOOGLE_CLIENT_ID`

any preference?

Reply:
AB? 🤔

I'd say backstage etc is redundant; AUTH_GOOGLE_x is fine